frontend/src/components/GroupCard/GroupCard.tsx

```diff
@@ -0,0 +1,72 @@
+import { H4, P } from "@expo/html-elements";
+import { ViewStyle } from "@expo/html-elements/build/primitives/View";
+import { Image } from "expo-image";
+import { Skeleton } from "moti/skeleton";
+import { StyleProp, StyleSheet, TouchableOpacity, View } from "react-native";
+
+import { useResponsiveLayout } from "@/hooks/useResponsiveLayout";
+import { useTypographyStyles } from "@/hooks/useTypographyStyles";
+import { Group } from "@/api/apiSchemas";
+
+interface GroupCardProps extends Partial<Group> {
+  isLoading?: boolean;
+  style?: StyleProp<ViewStyle>;
+}
+
+export const GroupCard: React.FC<GroupCardProps> = (props) => {
+  const styles = useStyles();
+  const { h4, bodySmall } = useTypographyStyles();
+  const blurHash = "LBQ0w8og~Vt7xufQWCfQ_1j[9Hay";
```

Comment:
Do we save the hash value here?

Reply:
what do you mean? if you mean extracting it for reuse in other files, we could but it's only being used in 1 file right now

Reply:
Yes, there is 1 file right now. This is the hash value to be used on other pages too? If yes, then would it be better for you to set it up now? The others including me could be not sure where to put it.

Reply:
sure thing